core/templates/components/shared-component.module.ts

```diff
@@ -38,99 +38,100 @@ import { NgBootstrapModule } from 'modules/ng-boostrap.module';
 import { DragDropModule } from '@angular/cdk/drag-drop';
 
 // Components.
-import { AudioBarComponent } from 'pages/exploration-player-page/layout-directives/audio-bar.component';
-import { DeleteAnswerGroupModalComponent } from 'pages/exploration-editor-page/editor-tab/templates/modal-templates/delete-answer-group-modal.component';
-import { ExplorationEmbedButtonModalComponent } from './button-directives/exploration-embed-button-modal.component';
-import { BackgroundBannerModule } from './common-layout-directives/common-elements/background-banner.module';
+import { AddHintModalComponent } from 'pages/exploration-editor-page/editor-tab/templates/modal-templates/add-hint-modal.component';
```

Comment:
Sort the imports based on the module it is imported from and not what is being imported here. From the diff it is a but difficult for me to judge the new components added here. Will it be possible for you reply with the new components you added to this file?

Reply:
The only new component I added to this file is the PostChapterRecommendationsComponent